src/Concordium/Client/Runner.hs

```diff
@@ -2121,6 +2147,7 @@ getContractUpdateTransactionCfg backend baseCfg txOpts indexOrName subindex rece
   txCfg <- getRequiredEnergyTransactionCfg baseCfg txOpts
   namedContrAddr <- getNamedContractAddress (bcContractNameMap baseCfg) indexOrName subindex
   contrInfo <- withClient backend . withBestBlockHash Nothing $ getContractInfo namedContrAddr
+  _ <- checkAndGetContractReceiveName contrInfo receiveName
   let namedModRef = NamedModuleRef {nmrRef = CI.ciSourceModule contrInfo, nmrNames = []}
   let contrName = CI.getContractName contrInfo
   schema <- withClient backend . withBestBlockHash Nothing $ getSchemaFromFileOrModule schemaFile namedModRef
```

Comment:
(I cannot comment on the line below)

The line below, shouldn't we use `updatedReceiveName` there as well, for the same reasons this is used in `invokeContract`?

Reply:
In the `invokeContract` the client is supposed to do the fallback receiveName resolution which is supposed to be done by the node here. Isn't it?

Reply:
After discussing it with Cem, I made the changes you requested, Ales.